docs/pages/versions/unversioned/sdk/task-manager.mdx

```diff
@@ -16,6 +16,7 @@ Some features of this module are used by other modules under the hood. Here is a
 
 - [Location](location.mdx)
 - [BackgroundFetch](background-fetch.mdx)
+- [Notification](notification.mdx)
```

Comment:
Hello @trajano, nice catch, thank you for the correction! 👍 

```suggestion
- [Notifications](notifications.mdx)
```

We use plural form for package, page and file name. Let's correct it to avoid 404.

Also, can I ask you to backport those changes to the other versioned docs?
* https://github.com/expo/expo/tree/main/docs/pages/versions

Reply:
I did only up to 48 since that's the one I am presently using.  I can't confirm 47 or below

Reply:
That's great, thanks for updates! 👌